lib/puppetserver/ca/action/migrate.rb

```diff
@@ -18,8 +18,9 @@ class Migrate
 
 Description:
   Migrate an existing CA directory to #{PUPPETSERVER_CA_DIR}. This is for
-  upgrading from Puppet Platform 6.x to Puppet 7. Use the currently configured
-  puppet.conf file in your installation, or supply one using the `--config` flag.
+  upgrading from Puppet Platform 6.x to Puppet 7. Uses the default puppet.conf
+  in your installation, override by supplying the `--config` flag.
```

Comment:
maybe, if we wanna be super duper explicit, we could say:
```suggestion
  in your installation, or use a different config by supplying the `--config` flag.
```
?